SPECIFICATION.md

```diff
@@ -224,13 +234,23 @@ parameters:
 2. Global letter spacing (`letter_spacing` from header)
 3. Leading space (leftmost X-coordinate) of second letter
 
+### Line Spacing
+
+Line spacing is the vertical distance between the lines of a multiline text. It
+can be defined with the `line_spacing` key in the header. The value basically
+depends on the height of ascenders and descenders to avoid vertically
+overlapping characters. So the `line_spacing` value should be set to a value
+slightly greater than the addition of the highest ascender and highest descender
+height. For a typical font this may be around 7 (78% of font height).
```

Comment:
If we define the line spacing as baseline to baseline, then we would have to adjust this wording.

From Wikipedia (linked above):

> In both cases, with a 10 pt font the usual leading is 12 pt, creating a 10/12 ratio, which is considered single spacing. Some type designers recommend a leading between 120% and 145% of the font size.

So we could recommend values between 10.8 and 12.6.